src/jsdoc_transformer.ts

```diff
@@ -90,6 +90,10 @@ export function maybeAddHeritageClauses(
           type: heritage.parentName,
         });
       }
+      // In the heritage==null case, we silently have dropped an @implements.
+      // In issue 1072 I advocate that we should never drop an @implements here,
```

Comment:
I don't understand this comment. It sounds as if you advocate something, but then didn't act on it? Is that a follow up?

The way I read the code it suggests that we below fall through and do emit?